src/RealtimeClient.ts

```diff
@@ -337,6 +337,21 @@ export default class RealtimeClient {
   setAuth(token: string | null): void {
     this.accessToken = token
```

Comment:
Should we not assign the accessToken if it is expired? Otherwise we end with an expired token assigned but not pushed.

Reply:
yeah JWT checking should be first 😶‍🌫️  will tackle